lib/termineter/modules/brute_force_login.py

```diff
@@ -57,61 +58,51 @@ def __iter__(self):
 				yield password
 				password = self.dictionary.readline()
 			self.dictionary.close()
-		raise StopIteration
+		#raise StopIteration; seems to have broken brute forcing multiple user id's
```

Comment:
Yeah I'm pretty sure this is left over from the old Python 2.x days. The current `setup.py` file declares 3.4 as the oldest version that's still supported so this should be just fine to remove.

Reply:
Cut this out entirely in the latest commit after addressing change requests